src/main/webapp/scripts/controller-auth.js

```diff
@@ -0,0 +1,46 @@
+import { postVideoComment } from '/scripts/post-comment.js';
+import { videoId, commentElement } from '/scripts/view.js';
```

Comment:
combine this import with line 3

Reply:
Done. Wasn't aware of this rule in the style guide but now I do. Thanks for pointing it out :)